src/osx/cocoa/dirdlg.mm

```diff
@@ -132,18 +145,42 @@
 {
     int result = wxID_CANCEL;
 
-    if (returnCode == NSOKButton )
+    if ( returnCode == NSOKButton )
     {
         NSOpenPanel* oPanel = (NSOpenPanel*)panel;
-        SetPath( wxCFStringRef::AsStringWithNormalizationFormC([[oPanel filenames] objectAtIndex:0]));
```

Comment:
Please don't change this now, as I'm going to push my rebased version soon, but I wonder about this: is it correct that we "lost" `AsStringWithNormalizationFormC()` call in the new version? Does `[NSURL fileSystemRepresentation]` take care of this?